packages/uniforms-material/src/LongTextField.tsx

```diff
@@ -19,24 +20,29 @@ const LongText = ({
   type = 'text',
   value,
   ...props
-}: LongTextFieldProps) => (
-  <TextField
-    disabled={disabled}
-    error={!!error}
-    fullWidth
-    helperText={(error && showInlineError && errorMessage) || helperText}
-    inputProps={{ readOnly }}
-    label={label}
-    margin="dense"
-    multiline
-    name={name}
-    onChange={event => disabled || onChange(event.target.value)}
-    placeholder={placeholder}
-    ref={inputRef}
-    type={type}
-    value={value ?? ''}
-    {...filterDOMProps(props)}
-  />
-);
+}: LongTextFieldProps) => {
+  const theme = useTheme();
+  const themeProps = theme.props?.MuiTextField;
+
+  return (
+    <TextField
+      disabled={disabled}
+      error={!!error}
+      fullWidth={themeProps?.fullWidth ?? true}
+      helperText={(error && showInlineError && errorMessage) || helperText}
+      inputProps={{ readOnly, ...(themeProps?.inputProps ?? {}) }}
```

Comment:
Do you think this is the right order here? I doubt that someone will ever specify theme-wise `readOnly` flag, but I think we'd like to override it. Or even override it only if there's `true` set.

Also, there's no need to `?? {}` here.